packages/mdc-menu/foundation.js

```diff
@@ -555,7 +562,10 @@ class MDCMenuFoundation extends MDCFoundation {
    */
   open({focusIndex = null} = {}) {
     this.adapter_.saveFocus();
-    this.adapter_.addClass(MDCMenuFoundation.cssClasses.ANIMATING_OPEN);
+
+    if (!this.quickOpen_) {
+      this.adapter_.addClass(MDCMenuFoundation.cssClasses.ANIMATING_OPEN);
```

Comment:
We should probably also conditionalize the `setTimeout` code below (and same for within `close`), although it's effectively a no-op in this case right now.